maap/utils/JobUtils.py

```diff
@@ -0,0 +1,18 @@
+# Valid job statuses (loosely based on OGC job status types)
```

Comment:
Can we name this file in snake case? `job_utils.py`

Reply:
On that note, in a _different_ PR, I want to deprecate all of the camelcase functions and use snake case, as that's the norm in Python. I opened an issue for this ages ago: #82

Reply:
Changed filename to `job_utils.py`